### PR TITLE
added new notation

### PR DESCRIPTION
**Pull request by lisa-wm**
added new eval notation proposed for latex-math

### CODE REVIEW

**Comment by lisa-wm** [comment]
[Add new eval notation to latex-math & check eval/tuning cheatsheet](https://trello.com/c/tmTOYq8C/108-add-new-eval-notation-to-latex-math-check-eval-tuning-cheatsheet)

File: slides/evaluation/slides-evaluation-generr.tex [hunk 1]
```diff
@@ -120,7 +120,7 @@
       loss $L$ or metric $\rho$.
   \item Nomenclature: ERM and \textbf{inner loss}; evaluation and \textbf{outer loss}. 
           % In this case, it holds by the law of the large numbers for the empirical risk $\riske$ that
-  % $$\E_{\D\sim(\Pxy)^{\ntrain}}\left[\frac{1}{n}\sumin L\left(\yi, \fhDlam(\xi)\right)  \right] \xrightarrow{n \to \infty} \GE(\inducer, \lamv, \ntrain, \rhoL) .$$
+  % $$\E_{\D\sim(\Pxy)^{\ntrain}}\left[\frac{1}{n}\sumin L\left(\yi, \fhDlam(\xi)\right)  \right] \xrightarrow{n \to \infty} \GE(\in, \lamv, \ntrain, \rhoL) .$$
```
Comment (ludwig-max) [inline slides/evaluation/slides-evaluation-generr.tex:123]: should be "\ind"? (Yes, is commented out, but if it's not deleted, it should be correct still)

File: slides/evaluation/slides-evaluation-generr.tex [hunk 2]
```diff
@@ -254,26 +254,26 @@
...
   Expectation \textbf{both} over $\Dtrain$ and $(\xv,y)$ independently from $\Pxy$.
  \item Retcon for GE of model: GE of learner, conditional on $\Dtrain$
-     $$\GE(\fh, L) := \GE(\inducer, \lamv, n_{\mathrm{train}}, \rho_L | \Dtrain) $$
-   if $\fh = \inducer(\Dtrain, \lamv)$ and $n_\mathrm{train} = |\Dtrain|$.
+     $$\GEfL := \GE(\ind, \lamv, n_{\mathrm{train}}, \rho_L | \Dtrain) $$
+   if $\fh = \ind(\Dtrain, \lamv)$ and $n_\mathrm{train} = |\Dtrain|$.
   % \item We must therefore estimate the generalization error, based on our 
   % train-test split:
-  % $$\GEh_{\Dtrain, \Dtest}(\inducer,
+  % $$\GEh_{\Dtrain, \Dtest}(\in,
```
Comment (ludwig-max) [inline slides/evaluation/slides-evaluation-generr.tex:273]: ind

File: slides/evaluation/slides-evaluation-generr.tex [hunk 3]
```diff
@@ -254,26 +254,26 @@
...
   % \lamv, \ntrain, \rho) =
   % \rho \left( \yv_{\Dtest}, \F_{\Dtest, 
-  % \inducer(\D_{\mathrm{train}}, \lamv)} \right).$$
+  % \in(\D_{\mathrm{train}}, \lamv)} \right).$$
```
Comment (ludwig-max) [inline slides/evaluation/slides-evaluation-generr.tex:276]: ind

**Review by ludwig-max — CHANGES_REQUESTED** [review]
3 times "ind" instead of "in"

**Pull request merged**